Dockerfile

```diff
@@ -0,0 +1,15 @@
+# use Docker's provided python image
+FROM python:2.7
+MAINTAINER natebrennand <natebrennand@gmail.com>
+
+# install all packages
+COPY ./config/requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
+
+# add the application directories
+ADD ./density /density
+WORKDIR /density
+
+# expose the port and start the server
+EXPOSE 5000
+CMD gunicorn density
```

Comment:
do we want to keep the gunicorn logs? currently we're using `gunicorn density:app -b 127.0.0.1:4000 --debug --log-file ../../logs/gunicorn.log --log-level debug &`